### PR TITLE
[APB-2044][MP] Upgrading to bootstrap-play-25

### PR DESCRIPTION
**Pull request by mariannepearson0**

### CODE REVIEW

**Comment by mariannepearson0** [comment]
Should StrictlyEnsureIndexes trait be removed as it is not used anywhere?

**Comment by arturopala** [comment]
`StrictlyEnsureIndexes` should be added to the repository traits

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
-
-resolvers += "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/"
+resolvers ++= Seq(
+  Resolver.url("hmrc-sbt-plugin-releases",
+    url("https://dl.bintray.com/hmrc/sbt-plugin-releases"))
+  (Resolver.ivyStylePatterns),
+  "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/"
+)
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "1.7.0")
```
Comment (johnrevill) [inline project/plugins.sbt]: Catalogue mentions 1.8.0 is latest version of sbt-auto-build plugin (https://catalogue.tax.service.gov.uk/service/agent-fi-relationship)

Comment (mariannepearson0) [reply project/plugins.sbt]: :+1: , what about the other items that are minorly behind in this catalogue? Shall I upgrade those as well?

**Pull request merged**